Felienne/hedy
# Removes turtle timeout

**Description**
We remove the timeout on running when using the turtle. Actually we don't remove it, but just set it to 10 minutes which should always be enough. Might be a good idea to perform a bit of clean-up here as we also have a timeout of 5 minutes on any level lower than 7 (as there are no loops yet) as well as a timeout of 5 seconds in all other cases.

**Fixes**
This PR fixes #2834.

**How to test**
Run a turtle program on a level higher than 7. Verify that the program does not timeout after 5 seconds.
